docs/understanding-tracking-design/managing-tracking-scenarios-console/index.md

```diff
@@ -0,0 +1,182 @@
+
+Implementation rules allow defining constraints and conditions on basic fields, based on the underlying schema properties of a particular tracking event. These rules ensure that the event data complies with the data structure's underlying schema, so when the scenario is triggered and flows downstream through the pipeline, it is validated against the schema.
+
+Add multiple implementation rules by clicking the **'+ Add implementation rules'** button in the **Implmentation rules** section. A new modal view will appear.
+
+This modal displays a list of rules for specific properties that can be set. This list of posible rules is populated based on the Data Structure properties from the specific version that the Tracking Scenario was created from. A search bar will also be available to help search and filtering by rule name.
+
+![](images/ts-7.png)
+
+You can click on any of the properties that are shown in the list. A new view inside the modal will show where you'll be able to configure the rule's **Type of condition**, **Value** and **Comments** properties. 
+
+![](images/ts-8.png)
+
+Click **'Add rule'** to add rules or **'Cancel'** button to discard changes.
+
+Once you've configured the rules, click the **'Save and update event'** to confirm the changes or **'Close'** to discard them.
+
+You will see a list of rules in the tracking detail view where you can edit or remove them.
+
+![](images/ts-9.png)
+
+### Entities
+
+Entities can be defined as part of the context for a tracking scenario, providing additional information about specific events such as clicks, conversions, impressions, payment info, and more. By including these entities, you can enrich the tracked data and gain deeper insights into user behavior.
+
+Add multiple entites by clicking the **'Add entities'** button. A new modal view for configuring entites will appear.
+
+![](images/ts-10.png)
+
+Similar to the **Add implementation rules** modal, you will be able to add multiple entities from the list of entites displayed, that can be sourced from your [Iglu](/docs/understanding-your-pipeline/glossary-of-terms/index.md#iglu) or custom user defined entites. For each of the entities, you can view detailed information about the properties of the entity by clicking **'View properties'**. A search bar will also be available to help search and filter by entity name.
+
+Once you've selected the entities you want to add to the tracking scenario, click the **'Select entities'** button to confirm or the **'Cancel'** button to discard your selections.
+
+Once selected you will be presented with a new view inside the modal where you can select the cardinality of the entities.
+
+![](images/ts-11.png)
+
+It will save the entites on **'Save entities'** click or back to the previous selection view clicking **'Back'**.
+
+Once the entities are saved, you will see a list of entities in the tracking detail view where you can edit, add or remove them.
+
+![](images/ts-12.png)
+
+* * *
```

Comment:
Not sure we need these... We are not using them on other pages. (But I don’t feel strongly about it.)

Reply:
Removed triple-start in the hole document.